### ISSUE TITLE
Scanner raises `Missing translations` if scope is a variable in `v1.0.6`

### ISSUE THREAD

**Issue by tim-s-ccs** [glebm/i18n-tasks#441]
I was updating to use this latest version of this gem (`v1.0.6`) and it raised some `Missing translations` where it had not done previously.

The areas where it raised this are where we are using a variable to hold the value of the scope (as not to repeat ourselves).

Below is an example of what my code looks like and the results of running the `health` task:
```
def message(state)
  case state
  when 'a'
    t('a', scope: MESSAGE_TRANSLATION_SCOPE)
  when 'b'
    t('b', scope: MESSAGE_TRANSLATION_SCOPE)
  when 'c'
    t('c', scope: MESSAGE_TRANSLATION_SCOPE)
  end
end

MESSAGE_TRANSLATION_SCOPE = 'path.in.translation.file'.freeze
```
_The message that is raised_:
```
+--------+------+-----------------------------------+
| Locale | Key  | Value in other locales or source  |
+--------+------+-----------------------------------+
|  all   | a    | app/path/to/file.rb:4             |
|  all   | b    | app/path/to/file.rb:6             |
|  all   | c    | app/path/to/file.rb:8             |
+--------+------+-----------------------------------+
```

I understand that this is a static analysis tool so I'm not expecting it to know what the value of the variable is. However, as previously stated, there were no issues being raised in these files on `i18n-tasks <= v1.0.5`. I was able to do some debugging and narrowed it down to the changes done in https://github.com/glebm/i18n-tasks/commit/0a2f94926969766a410a854d2c868208175e56fd.

I guess my question is twofold.

1. What were the changes in this commit that caused this change in behaviour
2. Is there any changes I can make to prevent seeing this error (other than adding to `ignore-missing` in the config)

**Comment by glebm** [comment]
You do have `strict: false` correctly set in the config, right?

https://github.com/glebm/i18n-tasks/blob/fed0923ea20864f39e8bf93ec8384164b08c18ba/templates/config/i18n-tasks.yml#L85-L86

/cc @davidwessman

**Comment by tim-s-ccs** [comment]
Thanks for looking at this. I can confirm that my issue is fixed in `v1.0.7`
